README.md

```diff
@@ -4,3 +4,22 @@ AzureIntegration
 Features that integrate ASP.NET Core with Azure.
 
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+
+SiteExtensions
+===
+
+To install a nightly preview of the ASP.NET Core runtime site extension for testing purposes:
+1. In the Azure portal select App Services -> your site -> Application settings
+1. Set `SCM_SITEEXTENSIONS_FEED_URL` application setting to `https://dotnet.myget.org/F/aspnetcore-release/`
+1. Go to `Advanced Tools` -> `Site extensions` -> `Gallery`
+1. Enter `AspNetCoreRuntime` into `Search` box and click `Search`
+1. Click `+` to install site extension, click `Restart site` on the right side of the page when installation finishes.
+1. Restart site in `Overview` tab of `App service`
+
+
+To update ASP.NET Core runtime site extension:
+1. Stop site in `Overview` tab of `App service`
+1. Go to `Advanced Tools` -> `Site extensions`
```

Comment:
Can also click on `DEVELOPMENT TOOLS` -> `Extensions` -> `ASP.NET Core Runtime Extensions` to open a slice for the extension. `Update` link on the top left of that slice will be available (undimmed) after a new version comes out. (Going through `Advanced Tools` only really helps when you need to search. Unfortunately, `Extensions` -> `Add` opens a slice that loads slowly and then doesn't display much information.)